packages/rsocket-core/src/index.ts

```diff
@@ -23,3 +23,4 @@ export * from "./RSocket";
 export * from "./RSocketConnector";
 export * from "./RSocketServer";
 export * from "./Transport";
+export * from "./ClientServerMultiplexerDemultiplexer";
```

Comment:
I'm a little concerned about the size of the API surface we expose from `rsocket-core` currently. We can/should review what is exposed by the API before we publish a dependable version.

Having a broad API surface might be unavoidable since other packages in the monorepo rely on many bits from `rsocket-core`, however it could make maintenance difficult if we have to increment major versions too often if/when what are essentially internal APIs/classes are modified.

Reply:
This is rather an internal class and should not be used by users. I don't know if we can bring needed isolation without copy/pasting classes here and there

Reply:
alternatively, we can make a transport common module which will be exposing this internal classes somehow